src/containers/Devices/Devices.jsx

```diff
@@ -19,6 +19,10 @@ class Devices extends React.Component {
     this.handleAddDeviceClose = this.handleAddDeviceClose.bind(this)
   }
 
+  componentDidMount () {
+    this.props.fetchDevices()
+  }
+
   handleAddDeviceClick () {
     this.setState({ addDevice: true })
```

Comment:
Info : if there is no signalhub, when I click on Add device button, there is no error message.  I only see the modal for a few seconds and it disappears.
Can you test it please ?

Reply:
Good catch, I fixed the notification position